src/iexamine.cpp

```diff
@@ -974,23 +974,35 @@ void iexamine::slot_machine(player *p, map*, const tripoint&)
 void iexamine::safe(player *p, map *m, const tripoint &examp)
 {
     if (!p->has_amount("stethoscope", 1)) {
-        add_msg(m_info, _("You need a stethoscope for safecracking."));
-        return;
+        if (query_yn(_("The safe is locked. Input a random combination?"))) {
+            p->moves -= 100;
+            // one_in(30^3) chance of guessing
+            if (one_in(27000)) {
```

Comment:
I would think that it's obvious that you shouldn't try to guess a combination safe's code. Maybe it needs a "You twiddle the dial for a moment." message instead, with a "You twiddle the dial for a moment.. and miraculously, it works!" success.